sourcegen-model/src/main/java/io/micronaut/sourcegen/model/ExpressionDef.java

```diff
@@ -58,10 +54,12 @@ public sealed interface ExpressionDef
     permits ExpressionDef.ArrayElement, ExpressionDef.Cast, ExpressionDef.ConditionExpressionDef, ExpressionDef.Constant, ExpressionDef.GetPropertyValue, ExpressionDef.IfElse, ExpressionDef.InstanceOf, ExpressionDef.InvokeGetClassMethod, ExpressionDef.InvokeHashCodeMethod, ExpressionDef.InvokeInstanceMethod, ExpressionDef.InvokeStaticMethod, ExpressionDef.MathBinaryOperation, ExpressionDef.MathUnaryOperation, ExpressionDef.NewArrayInitialized, ExpressionDef.NewArrayOfSize, ExpressionDef.NewInstance, ExpressionDef.Switch, ExpressionDef.SwitchYieldCase, VariableDef, Lambda {
 
     /**
-     * Get the operands that the expression applies to.
-     * @return The operands
+     * All the expressions that are represented by this statement.
```

Comment:
The javadoc needs fixing.

Also, should the method be nestedExpressions or something that implies that the stream doesn't include this expression?